### ISSUE TITLE
Buttons are not available in taskbar

### ISSUE THREAD

**Issue by DiptangshuRoy** [evilpro/Taskplay#20]
Buttons are available inside hidden icons instead of taskbar, where it should be.

![image](https://github.com/user-attachments/assets/aa5e0263-c3e5-4be1-81b4-1a181857335f)

**Comment by aspauldingcode** [comment]
Do you think this happens when there's too many pinned apps?

**Comment by DiptangshuRoy** [comment]
> Do you think this happens when there's too many pinned apps?

No, I checked by removing some apps from there & sorry for late reply:)
